src/style.css

```diff
@@ -1 +1,83 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  display: flex;
+  height: 60px;
+  justify-content: space-between;
+  font-size: 12px;
+  text-transform: uppercase;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo_pic {
+  width: 40px;
+  height: 40px;
+  margin: 10px 0 10px 50px;
```

Comment:
Do you really need to add margins for logo? It can just be a padding of header.
You can also try to add sizes for the link, not logo